.github/workflows/update_chromium_revision.yml

```
@@ -0,0 +1,37 @@
+name: update_chromium_revision
+on:
+  # Trigger every day at 10 UTC, or manually.
+  schedule:
+   - cron: 0 10 * * *
+  workflow_dispatch:
+jobs:
+  update:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+        cache: 'pip'
+    - name: Install dependencies
+      run: pip install requests
+    - name: execute update_chromium_revision.py
+      run: python tools/wpt/update_chromium_revision.py
+    - name: Create pull request
+      # Use a conditional step instead of a conditional job to work around #20700.
+      if: github.repository == 'web-platform-tests/wpt'
+      uses: peter-evans/create-pull-request@v4
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        author: wpt-pr-bot <wpt-pr-bot@users.noreply.github.com>
+        title: "Update latest available Chromium revisions"
```

Can you update the commit/PR wording to match recent changes? For this line, maybe:

```suggestion
        title: "Update pinned Chromium revision"
```

I think the commit message should also be the same as the PR title.